.github/workflows/upstream-build.yml

```diff
@@ -57,11 +57,19 @@ jobs:
         working-directory: zephyr-silabs
         shell: bash
         run: |
-          west twister -v --inline-logs \
-            -s drivers.dma.chan_blen_transfer \
-            -p siwx917_rb4338a \
-            -x DTC_OVERLAY_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.overlay \
-            -x EXTRA_CONF_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.conf
+          west twister -s drivers.dma.chan_blen_transfer -p siwx917_rb4338a -x \
+            DTC_OVERLAY_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.overlay \
+            -x EXTRA_CONF_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.conf \
+            -v --inline-logs
+
+      - name: Build Scatter-Gather DMA test
+        working-directory: zephyr-silabs
+        shell: bash
+        run: |
+          west twister -s drivers.dma.scatter_gather -K -p siwx917_rb4338a -x \
```

Comment:
And here. Just to be clear: I'm asking to make the line split before `-x` instead of after it.